starknet_devnet/blocks.py

```diff
@@ -306,3 +306,22 @@ async def store_pending(
     def get_state(self, block_number: int) -> StarknetState:
         """Return state at block with `number`"""
         return self.__state_archive.get(block_number)
+
+    async def abort_block_by_hash(self, block_hash: str) -> str:
+        """
+        Abort block by given block hash.
+        """
+        numeric_hash = _parse_block_hash(block_hash)
+
+        if numeric_hash in self.__hash2num:
+            block_number = self.__hash2num[int(block_hash, 16)]
+            block = await self.get_by_number(block_number)
+
+            block_dict = block.dump()
+            block_dict["status"] = BlockStatus.ABORTED.name
+            block_dict["transaction_receipts"] = None
+            self.__num2block[block_number] = StarknetBlock.load(block_dict)
+
+            return block.block_hash
+
+        return ""
```

Comment:
What is this?

Reply:
I wanted to return an empty string if condition `if numeric_hash in self.__hash2num` is not met but it can be `None` so I removed it here 9c613510e84280d0ffab641bab272a84c93c05ca

Reply:
this `return ""` is outdate

we are testing the unhappy path in `test_abort_not_existing_block()` test